### DISCUSSION TITLE
Error while extracting brand

### DISCUSSION THREAD

**Discussion by joesong168** [libvips/libvips#3187] [Q&A]
## Questions, enhancements, tips, etc.

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report

**Describe the bug**

**To Reproduce**
1. Use Image "https://viewers-dev.etu.wiki/64cd43c31f68c241b974f3101bf4fed6.tif"
2. Use Configuration "vips extract_band 64cd43c31f68c241b974f3101bf4fed6.tif 64cd43c31f68c241b974f3101bf4fed6.v 0 --n 4"
3. See error  (vips.exe:27456): VIPS-WARNING **: 23:04:35.608: error in tile 0 x 8262 source input: Using code not yet in table

**Expected behavior**
band extracted

**Actual behavior**
error in console

**Screenshots**
![截屏2022-11-23 23 11 40](https://user-images.githubusercontent.com/10493864/203581407-a6397f98-dc89-4798-8fbf-dd7b4d10e8d4.png)

**Environment**
(please complete the following information)
 - OS: Windows 10
 - Vips: vips-8.13.2
 - 

**Additional context**

**Comment by jcupitt** [comment]
Hello @joesong168,

I tried with the official `tiffcp` program (part of libtiff):

```
$ tiffcp 64cd43c31f68c241b974f3101bf4fed6.tif x.tif
TIFFFetchNormalTag: Warning, Incorrect value for "RichTIFFIPTC"; tag ignored.
test.tif: Using code not yet in table.
test.tif: Error, can't read strip 486.
```

So I think your image is corrupt.

**Comment by joesong168** [comment]
However, the file could be opened and edited in photoshop. Is it possible to tolerate this kind of corruption?

**Comment by jcupitt** [comment]
Where did you get file file from? Did you make it with photoshop? It is possible that it has been truncated?

I think I would post to the libtiff mailing list:

https://lists.osgeo.org/mailman/listinfo/tiff

Include:

1. your test image
2. what happens when you try to copy the file with `tiffcp`
3. say that photoshop (say which version) can open the file successfully
4. which libtiff version you tested with (I used 4.4.0)

**Comment by joesong168** [comment]
Thanks a lot, I will consider it.
